packages/core/src/public_api.ts

```diff
@@ -11,4 +11,5 @@ export {
 export * as ɵcomponents from './utils/expose-component';
 export * as ɵtypes from './utils/types';
 export type { KnownModelIds } from './utils';
+export type { AzureKnownModelIds } from './utils';
```

Comment:
Any issues exporting this?

Reply:
@hb-coding no, sir. it's been updated as part of the docs/api updates we pushed. rebase from `main` and this should be fixed. sorry we did not reply to this sooner 😬